Packs/CheckPointNDR/Integrations/CheckPointNDR/README.md

```diff
@@ -37,46 +37,9 @@ Retrieve all NDR Insights
 
 | **Argument Name** | **Description** | **Required** |
 | --- | --- | --- |
-| from | Date and time from which to fetch insights. Default is Last 24 hours.| Optional | 
-| create_events | If true, the command will create events, otherwise it will only display them. Possible values are: true, false. Default is false. | Optional | 
```

Comment:
I see that the command does not exist at all, neither in the yml file nor in the integration
If there is indeed no such command, you can delete all its documentation starting from line 22